tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -283,6 +283,35 @@ def _grid_sample_input_wrangler(
     return args, kwargs
 
 
+def _im2col_input_wrangler(
+    args: list[Any], kwargs: dict[str, Any]
+) -> tuple[list[Any], dict[str, Any]]:
+    # Move kernel_size, dilation, padding and stride from args to kwargs
+    if len(args) == 5:
```

Comment:
Is this guaranteed? Or, is it possible that the caller can mix args and kwargs?

Reply:
This depends on how the test data is constructed and that tends to be relatively stable

Reply:
This is correct